src/Extension/Task/VcsCloneTask.php

```diff
@@ -103,7 +103,7 @@ public function getHelp(): ?string
      */
     public function getId(): string
     {
-        return 'sdk:git:clone';
+        return 'sdk:vcs:clone';
```

Comment:
is it a functional requirement to have `sdk:vcs:*`? IMO, just `vcs:clone` looks better and more logical, because it is an extension feature, not a core functionality.